catalyst_voices_packages/catalyst_cardano_serialization/lib/src/types.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:cbor/cbor.dart';
+
+/// Specifies an amount of ADA in terms of lovelace.
+typedef Coin = int;
```

Comment:
Do you think we should consider using a new type pattern here?

Reply:
We need arithmetic operators (i.e. when calculating a fee) so either we unwrap the type when we do it or we expose these operators on this new type. I think we can expose them (add, subtract, multiply, divide)

Reply:
>  I think we can expose them (add, subtract, multiply, divide)

It would be great!

Reply:
Done